cfgov/jinja2/v1/_includes/molecules/featured-menu-content.html

```diff
@@ -1,6 +1,6 @@
 {# ==========================================================================
 
-   featured_menu_content
+   featured_menu_content.render()
 
```

Comment:
So we have some inconsistencies in this, or more specifically three patterns:
- **(a) A template that doesn't have a macro and is just straight HTML/Jinja with `value` variables.** In this case the full atomic name is used, like `Featured Menu Content`, since this template is straight-up included somewhere else.
- **(b) A template that has a macro that has a name that follows the file name, which spits out HTML.** In this case the function name is used as it would appear after it is imported elsewhere (w/o params), like `featured_menu_content()`.
-  **(c) A template that has a macro named "render" that spits out HTML.** In this case render is used along with the template name as it would appear after it is imported elsewhere (w/o params), like `featured_menu_content.render()`.

(b) and (c) are essentially overlapping patterns, except their imports differ like:

- (b) `from 'molecules/featured-menu-content.html' import featured_menu_content as featured_menu_content`.
- (c) `import 'molecules/featured-menu-content.html' as featured_menu_content`.

(c) was the original pattern and is more flexible I think, since it allows additional functions to be included in the template without a need to modify the base import name. For example, `atomic_thing.render()` and `atomic_thing.settings()`, versus `atomic_thing()` and `atomic_thing_settings()`.

In regard to this PR, it just updates the code comment to be consistent and does not change (c) to (b).